### DISCUSSION TITLE
Solutions for insufficient video memory in large-scale system calculations

### DISCUSSION THREAD

**Discussion by shisan-yx** [deepmodeling/dpgen#1535] [Q&A]
I would like to inquire about the issue of insufficient video memory when running molecular dynamics (MD) simulations with a system that has tens of millions of atoms, using the software LAMMPS for the MD stage. Could you please share some solutions that others have found effective。

**Comment by njzjz** [comment]
In this situation, MPI is often used across multiple GPU cards.
